src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/SQLitePersistorTest.java

```diff
@@ -94,7 +94,7 @@ public void write() throws IOException, SQLException, InterruptedException {
             71,
             "heap usage in percentage",
             60);
-    ResourceFlowUnit rfu = new ResourceFlowUnit(System.currentTimeMillis(), context, summary);
+    ResourceFlowUnit rfu = new ResourceFlowUnit(System.currentTimeMillis(), context, summary, true);
 
     Node rca = new TestRca();
```

Comment:
It might makes sense to add a test case that creates a graph which executes on data node and the master partly and then check if it is persisted at each places. 
We hit the bug because we didn't have a Unit test to validate this in the first place. Let's add one now